UI/user.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Maintenance Tracker | User</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
+</head>
+
+<body>
+    <div class="ch-wrapper">
+        <div class="ch-aside">
+            <div id="ch-nav">
+                <ul class="ch-nav ch-card-1">
+                    <li>
+                        <h2><a href="">Maintenance Tracker</a></h2>
+                    </li>
+                    <li class="right"><a href="index.html"><button class="ch-btn-sm" onclick="window.location.replace(index.html)">log out</button></a></li>
```

Comment:
I want to believe the index.html is the landing page which you have not yet worked upon